packages/cli/src/lib/api.ts

```diff
@@ -136,3 +136,19 @@ export function getReleases(heroku: APIClient, appId: string) {
     partial: true,
   })
 }
+
+export function getConfigVars(heroku: APIClient, pipelineID: string) {
+  return heroku.request<Heroku.ConfigVars>(`/pipelines/${pipelineID}/stage/test/config-vars`, {
+    method: 'GET',
+    headers: {Accept: PIPELINES_HEADER},
+  })
+}
+
+export function setConfigVars(heroku: APIClient, pipelineID: string, body: Heroku.ConfigVars) {
+  return heroku.request<Heroku.ConfigVars>(`/pipelines/${pipelineID}/stage/test/config-vars`, {
+    method: 'PATCH',
+    headers: {Accept: PIPELINES_HEADER},
+    path: `/pipelines/${pipelineID}/stage/test/config-vars`,
+    body,
+  })
+}
```

Comment:
Since these functions are specific to pipelines, and since they are now in a more general `api` file, should they include "pipelines" in the name somewhere? Such as `getPipelineConfigVars` and `setPipelineConfigVars`?